src/changelog.rs

```diff
@@ -10,6 +10,8 @@ use crate::*;
 ///  - Fixed [`fs::ReadDir`]: Implement `PinnedDrop` to poll future stored internally,
 ///    otherwise it would drop the internal request ids too early, causing read task to fail
 ///    when they should not fail.
+/// ## Added
+///  - Exposing some methods in [`Sftp`] to check available extensions
```

Comment:
Can you please list details of the added apis, e.g.

```suggestion
///  - Add new fn `Sftp::support_limits`
///  - Add new fn `Sftp::support_expand_path`
```